docs/cloud-native-security/get-started-with-kspm.asciidoc

```diff
@@ -0,0 +1,189 @@
+5. Select *EKS* from the *Kubernetes Deployment* menu. A new section for AWS credentials will appear.
+
+[discrete]
+[[kspm-setup-eks-auth]]
+==== Authenticate to AWS
+
+There are several options for how to provide AWS credentials:
+
+* <<kspm-use-keys-directly,Use access keys directly>>
+* <<kspm-use-temp-credentials,Use temporary security credentials>>
+* <<kspm-use-a-shared-credentials-file,Use a shared credentials file>>
+* <<kspm-use-iam-arn,Use an IAM role ARN>>
+
+Regardless of which option you use, you'll need to grant the following permissions:
+
+```
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "VisualEditor0",
+            "Effect": "Allow",
+            "Action": [
+                "ecr:GetRegistryPolicy",
+                "eks:ListTagsForResource",
+                "elasticloadbalancing:DescribeTags",
+                "ecr-public:DescribeRegistries",
+                "ecr:DescribeRegistry",
+                "elasticloadbalancing:DescribeLoadBalancerPolicyTypes",
+                "ecr:ListImages",
+                "ecr-public:GetRepositoryPolicy",
+                "elasticloadbalancing:DescribeLoadBalancerAttributes",
+                "elasticloadbalancing:DescribeLoadBalancers",
+                "ecr-public:DescribeRepositories",
+                "eks:DescribeNodegroup",
+                "ecr:DescribeImages",
+                "elasticloadbalancing:DescribeLoadBalancerPolicies",
+                "ecr:DescribeRepositories",
+                "eks:DescribeCluster",
+                "eks:ListClusters",
+                "elasticloadbalancing:DescribeInstanceHealth",
+                "ecr:GetRepositoryPolicy"
+            ],
+            "Resource": "*"
+        }
+    ]
+}
+```
+
+[discrete]
+[[kspm-use-keys-directly]]
+===== Option 1 - Use access keys directly
+Access keys are long-term credentials for an IAM user or the AWS account root user. To use access keys as credentials, you need to provide the `Access key ID` and the `Secret Access Key`.
+
+For more details refer to AWS's documentation for https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html[Access Keys and Secret Access Keys].
+
+IMPORTANT: You must select "Programmatic access" when creating the IAM user.
+
+[discrete]
+[[kspm-use-temp-credentials]]
+===== Option 2 - Use temporary security credentials
+Temporary security credentials can be configured in AWS to last for a specified period of time. They consist of an access key ID, a secret access key, and a security token, which is typically found using `GetSessionToken`.
+
+Because temporary security credentials are short term, once they expire you will need to generate new ones and manually update the integration's configuration to continue collecting cloud posture data. Update the credentials before they expire to avoid data loss.
```

Comment:
```suggestion
Because temporary security credentials are short term, once they expire, you will need to generate new ones and manually update the integration's configuration to continue collecting cloud posture data. Update the credentials before they expire to avoid data loss.
```